lib/Doctrine/ORM/EntityManager.php

```diff
@@ -148,9 +148,9 @@
      */
     protected function __construct(Connection $conn, Configuration $config, EventManager $eventManager)
     {
-        $this->conn              = $conn;
+        $this->conn          = $conn;
         $this->config            = $config;
-        $this->eventManager      = $eventManager;
+        $this->eventManager    = $eventManager;
```

Comment:
This looked funny, and looks even funnier now

Reply:
indeed indentation is broken here, not sure why this change

Reply:
seems this is just for test

Reply:
Oh ok